### PR TITLE
Fogl 1724

### PR DESCRIPTION
**Pull request by amarendra-dianomic**

### CODE REVIEW

File: python/foglamp/plugins/north/http_north/http_north.py [hunk 1]
```diff
@@ -50,17 +50,20 @@
     'shutdownWaitTime': {
         'description': 'Time in seconds, the plugin should wait for pending tasks to complete before shutdown',
         'type': 'integer',
-        'default': '10'
+        'default': '10',
+        'order': '3'
```
Comment (praveen-garg) [inline python/foglamp/plugins/north/http_north/http_north.py]: is it being used?

File: python/foglamp/plugins/north/http_north/http_north.py [hunk 2]
```diff
@@ -50,17 +50,20 @@
...
     },
     "applyFilter": {
         "description": "Should filter be applied before processing data",
         "type": "boolean",
-        "default": "False"
+        "default": "False",
```
Comment (praveen-garg) [inline python/foglamp/plugins/north/http_north/http_north.py]: should be lowercase?

**Review by praveen-garg — CHANGES_REQUESTED** [review]
there is one unused config item and still has higher order than others, also boolen true false should be lowercase

**Comment by amarendra-dianomic** [comment]
@praveen-garg seems that currently unused config item is set for future use in a TODO comment. Boolean value changed to lower case.

**Comment by praveen-garg** [comment]
there is no shutdown so remove that TODO comment and this config item please.

Though, We shall need connectionTimeout default to 10 secs and handle it with aiohttp context. but that can be done later. and also addCompression : true/ false; default to false

**Comment by amarendra-dianomic** [comment]
@praveen-garg feedback changes done. Pl check.

**Comment by praveen-garg** [comment]
where are you using compression and timeout?

File: python/foglamp/plugins/north/http_north/http_north.py [hunk 3]
```diff
@@ -87,12 +81,13 @@ def plugin_init(data):
 
 async def plugin_send(data, payload, stream_id):
     # stream_id (log?)
-    is_data_sent, new_last_object_id, num_sent = await http_north.send_payloads(payload)
-    return is_data_sent, new_last_object_id, num_sent
-
+    try:
+        is_data_sent, new_last_object_id, num_sent = await http_north.send_payloads(payload)
+        return is_data_sent, new_last_object_id, num_sent
```
Comment (praveen-garg) [inline python/foglamp/plugins/north/http_north/http_north.py]: else: 
 `        return is_data_sent, new_last_object_id, num_sent`

Comment (amarendra-dianomic) [reply python/foglamp/plugins/north/http_north/http_north.py]: done. PL check.

**Review by praveen-garg — APPROVED** [review]

**Pull request merged**